packages/react-components/src/LoadingContainer.js

```diff
@@ -29,6 +29,23 @@ class LoadingContainer extends Component {
       );
     }
 
+    if (this.props.web3.networkMismatch) {
+      if (this.props.networkMismatchComp) {
+        return this.props.networkMismatchComp;
+      }
+
+      return (
```

Comment:
This section can be re-written as:

```js
return this.props.networkMismatchComp || <main>...</main>
```

This allows us to avoid the nested if statement and also makes it clear that `this.props.networkMismatchComp` is a component.